students/BrianB/lesson05/except_exercise.py

```diff
@@ -42,13 +50,24 @@
 
 langs = ['java', 'c', 'python']
 
+# <<<<<<< master
+try:
+    more_joke = more_fun(langs[0])
+except IndexError:
+    pass
+
+# =======
 more_joke = more_fun(langs[0])
```

Comment:
This is already handled in your try/except above, I believe you meant to delete this?